test/general-tests/signature-argument/check.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+signature_len=6
+signature="1 2 3 4 5 6"
+
+# compile signatures contract
+npx hardhat starknet-compile contracts/signatures.cairo
+
+# deploy contract
+output=$(npx hardhat starknet-deploy --wait --starknet-network "$NETWORK" starknet-artifacts/contracts/signatures.cairo/)
+echo $output
+
+address=$(echo $output | sed -r "s/.*Contract address: (\w*).*/\1/")
+
+
+echo "Address: $address"
+
+# test call
+call_output=$(npx hardhat starknet-call \
+    --contract "signatures" --address "$address" \
+    --function "get_signature" --starknet-network "$NETWORK" \
+    --signature "$signature")
+
+echo $call_output
```

Comment:
Did you leave the log lines such as this one on purpose?

Reply:
Yes